### PR TITLE
Plugins and perform find api

### PR DESCRIPTION
**Pull request by Dhiraj1405**

### CODE REVIEW

**Comment by Dhiraj1405** [comment]
Closes #20

File: Perform find.md [hunk 1]
```diff
@@ -6,67 +6,38 @@ Fetches data from the Order Management System (OMS) entities. By making a GET re
...
-```
-{
-  inputFields: 
-  fieldList: 
-  entityName: 
-  noConditionFind: 
-  viewSize: 
-}
-```
-#### Example
```
Comment (dt2patel) [inline Perform find.md]: we have a video about how to use perform find, check if its relevant and attach it here if that makes sense.

File: Plugins/Display app version with @hotwax-app-version-info package.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Display app version with @hotwax/app-version-info Package
+
+### Overview
+
+The App version info package is a software package developed by HotWax Commerce that provides functionality to retrieve version information of an application. By using this package, HotWax fetches and displays the version number and other relevant details of the application, which can be useful for monitoring and displaying version information within the application's user interface.
```
Comment (dt2patel) [inline Plugins/Display app version with @hotwax-app-version-info package.md]:
```suggestion
The App version info package is a software package developed by HotWax Commerce that provides functionality to retrieve version information of an application. Using this package helps with monitoring and displaying version information within the application's user interface for troubleshooting errors and bugs faced on certain versions of an app.
```

File: Plugins/Manage app styling with @hotwax-apps-theme package.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Manage app styling with @hotwax-apps-theme Package
+
+### Overview
+
+The Apps theme package is a software library developed by HotWax Commerce that provides styling functionalities for applications. By using this package, HotWax enhances the visual appearance and user interface of applications.
```
Comment (dt2patel) [inline Plugins/Manage app styling with @hotwax-apps-theme package.md]:
```suggestion
The Apps theme package is a software library developed by HotWax Commerce to deploy a consistent set of styles for all apps from a centrally managed style package. Global styles and branding overrides are managed from here and then imported into all apps.
```

File: Plugins/Save files with FileSaver package.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Overview
+
+The FileSaver package is a software library designed to save files in web applications that generate files on the client-side. This package is used in HotWax Commerce apps to save files into user's system. 
+
+## Use Cases
+
+### Case 1: Exporting packed orders CSV from the Fulfillment App
+
+When retailers generate their shipping labels outside of HotWax, they need to export all packed orders once and share it with carrier partner to generate shipping labels and give tracking codes. The FileSaver package helps the user quickly save the file in their systems by giving it their desired name.
```
Comment (dt2patel) [inline Plugins/Save files with FileSaver package.md]:
```suggestion
When retailers generate their shipping labels outside of HotWax, they need to export all packed orders once and share it with carrier partner to generate shipping labels and give tracking codes. The FileSaver package helps the user quickly save the file in their systems with their desired name.
```

**Review by dt2patel — CHANGES_REQUESTED** [review]
Added comments and suggestions. Mostly looks good though

File: Perform find.md (hunk 1 above)
Comment (Dhiraj1405) [reply Perform find.md]: sure adding

**Comment by Dhiraj1405** [comment]
Made suggested changes. Please merge @dt2patel

**Pull request merged**